framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -402,7 +405,7 @@ public void run() {
   private static void runGenerateScript(DataSource datasource) {
     int exitCode = 0;
     String command = CWD + "/resources/" + datasource.src;
-    LOG.debug("Running command " + command);
+    LOG.info("Running command " + command);
```

Comment:
I will isolate the change to add more debugging and create another pull request. 

The original issue addressed in this is a hacky one :) as sudeesh pointed out and the usefulness is also questionable as the state of the drillbit at the time of failure is not accurately captured (drill cancels running queries if client terminates connection)